public/app/features/explore/state/actions.ts

```diff
@@ -423,6 +423,8 @@ export function processQueryErrors(
       return;
     }
 
+    console.log(response); // To help finding problems with query syntax
```

Comment:
Maybe nitpicking but console.error would be nicer

Reply:
Will fix